contracts/factory/src/contract.rs

```diff
@@ -97,6 +107,9 @@ pub fn try_update_config(
     if let Some(pair_code_id) = pair_code_id {
         config.pair_code_id = pair_code_id;
     }
+    if let Some(commission_rate) = commission_rate {
+        config.commission_rate = commission_rate;
+    }
```

Comment:
Add logic to update the commission_rate in PAIR_INFO in Pair contract.

Reply:
This exists in `try_update_configs` function starting on line 486.